plugins/node/opentelemetry-instrumentation-graphql/src/utils.ts

```diff
@@ -363,7 +367,10 @@ export function wrapFieldResolver<TSource = any, TContext = any, TArgs = any>(
 
     let field: any;
     let shouldEndSpan = false;
-    if (config.depth >= 0 && config.depth < depth) {
+    if (
+      (config.depth >= 0 && config.depth < depth) ||
+      config.ignoreFields.includes(path[path.length - 1])
```

Comment:
you created a map but you still didn't use it

Reply:
fixed